apis/slo/v1alpha1/nodemetric_types.go

```diff
@@ -23,8 +23,25 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+type AggregationType string
+
+const (
+	// max is not welcomed since it may import outliers
```

Comment:
do we have to mention `max` here?

Reply:
in order to explain the reason that why we don't add the `MAX` aggregate type temporarily.